.gitlab-ci.yml

```diff
@@ -60,6 +60,11 @@ deploy-gcp-staging:
     #- docker login -u "gitlab-ci-token" -p "$CI_BUILD_TOKEN" $CI_REGISTRY
     - export HELM_REPLICA_COUNT=$HELM_REPLICA_COUNT_STAGING
     - export SECRET_KEY_BASE=$SECRET_KEY_BASE_STAGING
+    - export GCP_DATABASE_HOST=$GCP_DATABASE_HOST_STAGING
```

Comment:
should it be DATABASE_HOST=$GCP_DATABASE_HOST_STAGING ? I keep GCP for 2nd part because it deploy to both heroku and k8s at the same time, for heroku using heroku database addon, and for k8s using a database VM instances